pkg/apis/kops/validation/validation.go

```diff
@@ -1220,6 +1221,14 @@ func validateNetworking(cluster *kops.Cluster, v *kops.NetworkingSpec, fldPath *
 		allErrs = append(allErrs, validateNetworkingGCP(cluster, v.GCP, fldPath.Child("gcp"))...)
 	}
 
+	if v.Kindnet != nil {
+		if optionTaken {
+			allErrs = append(allErrs, field.Forbidden(fldPath.Child("kindnet"), "only one networking option permitted"))
+		}
```

Comment:
We probably should be setting optionTaken here, but we're also missing it on LyftVPC and GCP.  I'll send a fix separately :-)